python/ray/_private/services.py

```diff
@@ -207,7 +203,14 @@ def find_redis_address(address=None):
     return redis_addresses
 
 
+def find_redis_address_or_die_respect_environment():
+    if "RAY_ADDRESS" in os.environ:
+        return None
```

Comment:
Why is it None? Shouldn't it return os.environ["RAY_ADDRESS"]?

Reply:
That will cause the same problem as before where `ray.init` is called with some address and `RAY_ADDRESS` is also set